src/main/java/frc/robot/RobotContainer.java

```diff
@@ -320,10 +313,11 @@ public void setupAuto() {
           m_revolver, m_shooter, m_turret);
       break;
     case DRIVE_FORWARDS:
-      /*m_autonomousCommand = new RunCommand(() -> m_driveTrain.arcadeDrive(0.5, 0.0), m_driveTrain).withTimeout(2)
-          .andThen(new RunCommand(() -> {
-            m_driveTrain.arcadeDrive(-0.1, 0.0);
-          }, m_driveTrain).withTimeout(0.1));*/
+      /*
+       * m_autonomousCommand = new RunCommand(() -> m_driveTrain.arcadeDrive(0.5,
+       * 0.0), m_driveTrain).withTimeout(2) .andThen(new RunCommand(() -> {
+       * m_driveTrain.arcadeDrive(-0.1, 0.0); }, m_driveTrain).withTimeout(0.1));
+       */
```

Comment:
Please revert this

Reply:
Resolved in commit f8935a4

Reply:
Still a tiny change on this line to be resolved.

Reply:
Resolved in commit 7dc5f38